corehq/apps/app_manager/management/commands/migrate_all_apps_to_cmitfb.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from django.core.management import BaseCommand, call_command
+
+from corehq.apps.app_manager.dbaccessors import get_app
+from corehq.apps.es.apps import AppES
+
+
+logger = logging.getLogger('app_migration')
+logger.setLevel('DEBUG')
+
+
+class Command(BaseCommand):
+    help = '''
+        Migrate any non-migrated apps
+    '''
+
+    def handle(self, *args, **options):
+        # Find all apps not yet using vellum case management
+        app_query = AppES().is_build(False).term('vellum_case_management', False) \
+                           .term('doc_type', 'Application').source(['domain', '_id'])
```

Comment:
I don't think that you need `domain` and you could also use `get_ids()`  to get a list of ids